RWHW03.R

```diff
@@ -0,0 +1,17 @@
+setwd(/Users/johnmoten/Desktop/BSDG30901)
+
+library(ggplot2)
+library(readxl)
+
+usstates<-read.excel("us-states.xlsx", sheet = 1)
+
+state <- usstates$state
+date <- usstates$date
+deaths <- usstates$deaths
+cases <- usstates$cases
+
+
+ggplot(usstates, aes(date, deaths)) +
+  geom_line(group = state) +
+  geom_smooth(group = 1, size = 3, method = "lm", se = FALSE) +
+  labs(title = "COVID Deaths over Time", x = "Time") +
```

Comment:
I had a similar set up for my graph and also initially tried to do a time series of US cases organized by state, but saw that it was "messy" because of so much data. So, I like your idea of having a trend line - which would be plotting the mean of the data for each date? - as a way to provide new insight into that much data.
In terms of code presentation, it was easy to follow and organized! Maybe next time you could include the code you tried to make work, but comment it out, so I can see what might have gone wrong. I only mention this because you said there were issues with adding a theme:)